net/management-api-v2/PostType.cs

```diff
@@ -33,6 +33,13 @@
             Name = "Article title",
             Codename = "title",
             ContentGroup = Reference.ByCodename("article-copy"),
+			DefaultValue = new TextElementDefaultValueModel
```

Comment:
Should use whitespace characters instead of tab characters. The three tabs will make this unpredictable from display point of view. Every browser renders a different number of whitespaces for a tab.

Reply:
yep, my bad, sorry for that... Notepad++ uses tabs over the spaces by default